cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-ui.adoc

```diff
@@ -0,0 +1,290 @@
+====
+
+. Click *Ok*.
+
+== Confirming successful account association
+
+. You will be brought back to the original window. You should now see your AWS account in the drop down menu. If you see your account, account association was successful.
+
+. Select the account.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-associate3.png[]
+
+. You will see the account role ARNs populated below.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-account-roles.png[]
+
+. Click *Next*.
+
+== Creating the cluster
+
+. For the purposes of this workshop make the following selections:
++
+.Cluster settings
++
+* Cluster name: **<pick a name\>**
+* Version: **<select latest version\>**
+* Region: **<select desired region\>**
+* Availability: **Single zone**
+* Enable user workload monitoring: **leave checked**
+* Enable additional etcd encryption: **leave unchecked**
+* Encrypt persistent volumes with customer keys: **leave unchecked**
+
+. Click *Next*.
+
+. Leave the default settings on for the machine pool:
++
+.Default machine pool settings
++
+* Compute node instance type: **m5.xlarge - 4 vCPU 16 GiB RAM**
+* Enable autoscaling: **unchecked**
+* Compute node count: **2**
+* Leave node labels blank
+
+. Click *Next*.
+
+=== Networking
+
+. Leave all the default values for configuration.
+
+. Click *Next*.
+
+. Leave all the default values for CIDR ranges.
+
+. Click *Next*.
+
+=== Cluster roles and policies
+For this workshop, leave *Auto* selected. It will make the cluster deployment process simpler and quicker.
+
+[NOTE]
+====
+If you selected a *Basic OCM role* earlier, you can only use manual mode. You must manually create the operator roles and OIDC provider. See the "Basic OCM role" section below after you have completed the "Cluster updates" section and started cluster creation.
+====
+
+=== Cluster updates
+* Leave all the options at default in this section.
+
+=== Reviewing and creating your cluster
+. Review the content for the cluster configuration.
+. Click *Create cluster*.
+
+=== Monitoring the installation progress
+* Stay on the current page to monitor the installation progress. It should take about 40 minutes.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-cluster-create.png[]
+
+== Basic OCM Role
+
+[NOTE]
+If you created an *Admin OCM role* as directed above *ignore* this entire section. The OCM will create the resources for you.
+
+If you created a *Basic OCM role* earlier, you will need to manually create two more elements before cluster installation can continue:
+
+* Operator roles
+* OIDC provider
+
+//To understand what these do, please see the ROSA with AWS STS Explained tutorial section. xref needed
+
+=== Creating Operator roles
+. A pop up window will show you the commands to run.
++
+image::cloud-experts-getting-started-rosa-deployment-detailed-ui-create-cmds.png[]
```

Comment:
![image](https://github.com/openshift/openshift-docs/assets/93653094/3a90e40d-870d-4ab4-a792-6e6782216355)
Latest screenshot indicate that ROSA CLI is default choice. Please correct it. Thanks